Polyhedron/demo/Polyhedron/Plugins/IO/Surface_mesh_io_plugin.cpp

```diff
@@ -0,0 +1,91 @@
+#include <CGAL/Three/Polyhedron_demo_io_plugin_interface.h>
+#include <CGAL/Three/Polyhedron_demo_plugin_interface.h>
+#include <fstream>
+
+#include <QApplication>
+#include <QObject>
+#include <QAction>
+#include <QMainWindow>
+#include <QDebug>
+#include "Scene_surface_mesh_item.h"
+#include "Scene_polygon_soup_item.h"
+#include "Messages_interface.h"
+
```

Comment:
I think that list of headers is too big.

* Why `<QApplication>`?
* Why `"messages_interface.h"`?